### PR TITLE
Feature v 1.0.0

### PR DESCRIPTION
**Pull request by simandebvu**
### Added 💻 
- Random Quotes Generated on page load in home tab.
- Boostrap Styling for Menu Items.
- Separate Module for Display. Single responsibility.
- Github Pages Live Link

### CODE REVIEW

File: src/modules/display.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    element.appendChild(blockQuote);
+    element.appendChild(paragraph);
+
+    return element;
+  };
+
+  const createMenuCard = (foodItems) => {
+    const cardsContainer = document.createElement('div');
+    cardsContainer.classList.add('d-flex', 'flex-wrap', 'justify-content-around');
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#Description
```
Comment (Forison) [inline src/modules/display.js]: * [x] Let us also get rid of this commented code

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Change required❌

# 🙋‍♂️ Shingi 

#### You have done really well on this one, the content of each tab is stored in a module and are very readable. However you are missing out on this requirement, kindly address to get this project approved🤗.

* [x] Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) 
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```


#### Happy coding 👨‍💻

#### Addo Forison

**Comment by simandebvu** [comment]
### Fixed 🧑‍🔧 
- Removed comments in code.
- Necessary run common commands added in package.json inside `scripts'

**Review by Torres-ssf — APPROVED** [review]
### Project Approved :+1: 
Hi @simandebvu  :hand: 
Great work on this project :clap: 
Good luck in the future one.

Happy coding :computer: 
**Sergio Torres**

**Pull request merged**